presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -384,6 +385,19 @@ public TaskManagerConfig setMaxPagePartitioningBufferSize(DataSize size)
         return this;
     }
 
+    @NotNull
+    public Integer getMaxPagePartitioningBufferCount()
+    {
+        return maxPagePartitioningBufferCount;
+    }
+
+    @Config("driver.max-page-partitioning-buffer-count")
```

Comment:
Add @ConfigDescription

Reply:
Sure. Added `@ConfigDescription("Maximum number of buffers used by repartitioning per driver")`